139_Client_Server/python_server.py

```diff
@@ -0,0 +1,63 @@
+import socket # Gives us #socket.socket() and socket.sock_stream()
+import threading # For multiple connections
+
+clients = []
+
+
+# Thread for each client
+def serve(client_connect, client_address, client_id):
+  global clients
+  try:
+    print(f"Serve {client_id}")
+    while(1):
+      client_connect.sendall(bytes('Write something!',encoding='utf8'))
+      data = client_connect.recv(80)
+      data = data.decode('utf-8')
+      print(f"Got from {client_id} : {data}")
+      if ((data=='quit') or (data=='Quit')):
+        break
+      else:
+        client_connect.sendall(bytes(f'You sent {str(data)}',encoding='utf8'))
```

Comment:
```suggestion
        client_connect.sendall(b'You sent %s' % data.encode('utf-8'))
```

See https://www.python.org/dev/peps/pep-0461/

Also the indentation should always be 4 spaces.